scripts/g.extension/g.extension.py

```diff
@@ -1534,15 +1534,15 @@ def install_extension_std_platforms(name, source, url, branch):
                 # get the module name: PGM = <module name>
                 with open(os.path.join(r, 'Makefile')) as fp:
                     for line in fp.readlines():
-                        if "PGM" in line:
+                        if 'PGM = ' in line or 'PGM=' in line:
```

Comment:
Should the left side probably be `'PGM =' in line` ?

Reply:
In the [aa7b425](https://github.com/OSGeo/grass/commit/aa7b42596c8da6717d5b1d22c7d79a8c2fbb82b4) I used regular expresion which catch cases:

```
PGM = r.learn.ml2
PGM=r.learn.ml2
PGM                                  = r.learn.ml2
PGM      =                   r.learn.ml2
....
```

Reply:
> Should the left side probably be `'PGM =' in line` ?

This checking if line (readed from Makefile) has `PGM =`  and then try parse it (module name).